dom.bs

```diff
@@ -8896,29 +8763,29 @@ these steps:
 <ol>
  <li>Let <var>s</var> be the empty string.
```

Comment:
This whole algorithm just says "start node" and "end node" a lot but should probably say "the context object's start node"/end node.

(Preexisting problem, though.)